src/app/pages/component-category-list/component-category-list.scss

```diff
@@ -8,27 +8,33 @@
   justify-content: center;
 }
 
-@media (max-width: 1795px) {
+@media (max-width: 2215px) {
   .docs-component-category-list-summary {
-    max-width: 1120px;
+    max-width: 1560px;
   }
 }
 
-@media (max-width: 1495px) {
+@media (max-width: 1895px) {
   .docs-component-category-list-summary {
-    max-width: 820px;
+    max-width: 1240px;
   }
 }
 
-@media (max-width: 1195px) {
+@media (max-width: 1575px) {
   .docs-component-category-list-summary {
-    max-width: 520px;
+    max-width: 920px;
   }
 }
 
-@media (max-width: 654px) {
+@media (max-width: 1255px) {
   .docs-component-category-list-summary {
-    max-width: 220px;
+    max-width: 600px;
+  }
+}
+
+@media (max-width: 694px) {
+  .docs-component-category-list-summary {
+    max-width: 280px;
   }
 }
 
```

Comment:
I wonder if it would look better if we centered the cards with `justify-content: center` in `.docs-component-category-list`.

I think you would also need to center the summary text, which looks like it would require `display: flex;
    flex-direction: column;` on the `app-component-category-list`

![image](https://user-images.githubusercontent.com/479713/86853585-9e69f800-c06b-11ea-82ff-6759532a9c06.png)